src/tribler-core/tribler_core/check_os.py

```diff
@@ -104,26 +104,25 @@ def should_kill_other_tribler_instances():
             sys.exit(0)
 
 
+def is_tribler_process(name):
+    """
+    Checks if the given name is of a Tribler processs. It checks a few potential keywords that
+    could be present in a Tribler process name across different platforms.
+    :param name: Process name
+    :return: True if pid is a Tribler process else False
+    """
+    name = name.lower()
+    keywords = ['tribler', 'python']
+
+    return any(keyword in name for keyword in keywords)
```

Comment:
You're going to kill **every** Python interpreter in the system with this :scream:

Reply:
I'm not sure:

https://github.com/Tribler/tribler/blob/1520b18b78d1083da7afa7374cd8aa00a9af9bfe/src/tribler-core/tribler_core/check_os.py#L79-L88

Reply:
Phew, false alarm. We still restrict id by the old pid.

Reply:
@ichorid anyway, thank you for this warning :)